### ISSUE TITLE
a.user-avatar on firefox (no avatar pic) drop down not reachable

### ISSUE THREAD

**Issue by lorsulic** [taigaio/taiga-front#1137]
After a fresh taiga intallation, the .navbar a.user-avatar has no top and bottom padding and no avatar. The result is, if you hover over your name (upper right corner), the dropdown appears, but the gap between a.user-avatar and the dopdown is to big to reach it, except you are superfast.

Seen on: 
taiga 3.0.0-stable
Mozilla Firefox 49
Mac OS Siera 10.12

**Comment by Xaviju** [comment]
Sorry but we cannot reproduce it. Can you attach a screenshot? 
In the navbar there should be a default avatar. Seems a more like a installation-related problem

**Comment by lorsulic** [comment]
I got a 404 on user-avatar-01.png from the host i2.wp.com. All the other local css and image stuff is working fine.
See the screenshot, I covered my name with the black bar. That's how it looks like, after I set an avatar, the problem is gone.

<img width="474" alt="screen shot 2016-10-14 at 10 38 06" src="https://cloud.githubusercontent.com/assets/2997492/19381473/0082a390-91fc-11e6-96ab-0c47bb77c652.png">
